BWBCoverBot/main.py

```diff
@@ -89,7 +93,7 @@ def verify_and_update_cover(isbn_13: str, archive_contents: ZipFile) -> None:
     return
 
 
-def parser_for_zip_with_isbns(cover_zip_path: str) -> None:
+def parser_for_zip_with_isbns(cover_zip_path: str, ol: OpenLibrary) -> None:
```

Comment:
Would it be helpful if this function returned an `int` to let the caller know how many covers were verified and updated?

Reply:
I did this, I hope in line with what you were thinking. :)